modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc

```diff
@@ -33,7 +33,7 @@ $ cosign generate-key-pair k8s://openshift-pipelines/signing-secrets
 +
 Provide a password when prompted. Cosign stores the resulting private key as part of the `signing-secrets` Kubernetes secret in the `openshift-pipelines` namespace.
 
-. In the {tekton-chains} configuration, disable the OCI storage, and set the task run storage and format to `tekton`. 
+. In the {tekton-chains} configuration, disable the OCI storage, and set the task run storage and format to `tekton`.
 +
 [source,terminal]
 ----
```

Comment:
Customers are not supposed to update configmap but TektonConfig. Again, we could just reference `Configuring Tekton Chains` section and list the necessary fields and their values here.

Reply:
Referenced, with no xref because of the ban. Note that links to "signing" and "configuring" are now provided immediately under this module as "additional resources"; this is the best I can do with the resrictions that OpenShift enforces.